programming_foundations/lean_fridge_app_final_proj/lean_fridge_setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from sqlalchemy import create_engine
+
+Base = declarative_base()
+
+class Stock(Base):
+    sqlite_autoincrement=True
+    __tablename__ = 'stock'
+    id = Column(Integer, primary_key = True)
+    name = Column(String(20), nullable = False)
+    total = Column(Integer, primary_key = True)
+
+
+class itemProperty(Base):
+    __tablename__ = 'itemProperty'
+    id = Column(Integer, primary_key = True)
+    name = Column(String(20), nullable = False)
+    dates = Column(String(10), nullable = False)
```

Comment:
column name should not be plural usually, especially when you only plan to store one value per row which is the case here i assume? 
also for date, for date, you should use the `Date` type available in sql alchemy instead of a string:
http://docs.sqlalchemy.org/en/rel_1_0/core/type_basics.html#sqlalchemy.types.Date